### PR TITLE
[pdfmake] Add QR code padding and document language

### PR DESCRIPTION
**Pull request by kryops**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
  - https://github.com/bpampuch/pdfmake/commit/b96131a79e0d16bc1b7173e084b6cc666c102bf6
  -  https://github.com/bpampuch/pdfmake/commit/732ef549ffbf0b69c9c7526df6e2004f4b61ea43
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.
  - only a patch release of the library (0.2.9)

### CODE REVIEW

File: types/pdfmake/interfaces.d.ts [hunk 1]
```diff
@@ -1310,9 +1310,11 @@ export interface ContentQr extends ContentBase, ForbidOtherElementProperties<"qr
      * Size in `pt` to fit the QR code in.
      *
      * As QR codes are square, this essentially sets both the width and height.
+     * pdfmake seems to shrink QR codes using certain fixed factors, so it might actually be smaller
+     * than the value given here.
```
Comment (andipaetzold) [inline types/pdfmake/interfaces.d.ts:1314]: I'm not sure about this change.
It feels strange to comment in a type library about the functionality of the library itself. Imo, this should be part of the library documentation. I'd either change this to a statement (not "pdfmake make SEEMS to shrink", but "pdfmake does xxx") or revert this change.

File: types/pdfmake/interfaces.d.ts [hunk 2]
```diff
@@ -1310,9 +1310,11 @@ export interface ContentQr extends ContentBase, ForbidOtherElementProperties<"qr
...
      *
      * Defaults to a heuristic depending on the pixel size of the QR code (see {@link version})
-     * so that every pixel is `5pt` large.
+     * so that every pixel is about `4.5pt` large.
```
Comment (andipaetzold) [inline types/pdfmake/interfaces.d.ts:1317]: According to the implementation it's 5px [the implementation uses 5px](https://github.com/bpampuch/pdfmake/blob/master/src/qrEnc.js#L755).

**Review by andipaetzold — CHANGES_REQUESTED** [review]
Type change looks good

I am not sure about the jsdoc change for the `fit` prop

File: types/pdfmake/interfaces.d.ts (hunk 1 above)
Comment (kryops) [reply types/pdfmake/interfaces.d.ts]: True, that was written more from a surprised/confused user's perspective 😅 

I removed the commit.

File: types/pdfmake/interfaces.d.ts (hunk 2 above)
Comment (kryops) [reply types/pdfmake/interfaces.d.ts]: Huh. You're right, I think I got confused when adding the padding in https://github.com/kryops/pdfmake-html-renderer/commit/1460bb46f9dc9619afe2cd93d79cdd550c18f228 that my QR code was wider than pdfmake's - but that is because I was comparing QR codes with a different amount of QR pixels.

Reverted by removing the commit.

**Review by andipaetzold — APPROVED** [review]
LGTM

**Comment by kryops** [comment]
Ready to merge
